dspace-api/src/main/java/org/dspace/ctask/general/RegisterDOI.java

```diff
@@ -118,8 +119,9 @@ private String register(Item item) {
         String doi = null;
         // Attempt DOI registration and report successes and failures
         try {
-            log.debug("Registering DOI with skipFilter = " + skipFilter);
-            doi = provider.register(Curator.curationContext(), item, skipFilter);
+            Filter filter = FilterUtils.getFilterFromConfiguration("identifiers.submission.filter.curation",
+                    new TrueFilter());
```

Comment:
Minor: could TrueFilter be made singleton?

Reply:
done